index.js

```diff
@@ -10,52 +10,12 @@
  * MIT License
  */
 
-var fingerprint = require('./lib/fingerprint.js');
-var isCuid = require('./lib/is-cuid.js');
-var pad = require('./lib/pad.js');
+import fingerprint from './lib/fingerprint';
+import createCuid from './lib/cuid';
+import isCuid from './lib/is-cuid';
 
-var c = 0,
-  blockSize = 4,
-  base = 36,
-  discreteValues = Math.pow(base, blockSize);
+const cuid = createCuid(fingerprint);
 
-function randomBlock () {
-  return pad((Math.random() *
-    discreteValues << 0)
-    .toString(base), blockSize);
-}
+export { fingerprint, isCuid };
```

Comment:
is rollup not complaining about mixing default and named exports for the cjs output? I thought it would. Just wondering if we should do this for the esm output only because this isn't well defined for cjs:

```
const cuid = require('@bugsnag/cuid')

const { isCuid } = cuid
```

Which `isCuid` is this referring to, the one on `export default cuid`, or the one from `export { isCuid }`?